src/App/Fossa/Analyze/LicenseAnalyze.hs

```diff
@@ -1,4 +1,4 @@
-module App.Pathfinder.Types (
+module App.Fossa.Analyze.LicenseAnalyze (
```

Comment:
Do you think it's worthwhile to write a comment here as to why we're keeping `LicenseAnalyzeProject` and moving it into its own module?

Reply:
Good question, and yes. I've done this.